esp/templates/program/modules/studentclassregmodule/catalog.html

```diff
@@ -153,6 +167,19 @@ <h2 style="text-align: center;">
 hideClassesOutOfGradeRange(student_grade);
 </script>
 
+<script type="text/javascript">
+$j("#grade_filter").change(function(){
+		var grade =  $j(this).find('option:selected').prop('value');
+    if(grade == "all"){
+        $j('.show_class').show();
+    }
+    else{
+    		$j('.show_class').hide();
```

Comment:
Can you fix the indentation here (not sure why it's wrong, perhaps a spaces/tabs issue?), and move `} else {` onto one line?  Having things inconsistently indented makes them really hard to read.